polaris-for-vscode/README.md

```diff
@@ -0,0 +1,23 @@
+# `polaris-for-vscode` Extension
+
+:warning: This extension is still an experimental prototype
+
+VSCode intellisense features to support developing with the Polaris Design System.
```

Comment:
```suggestion
VSCode features to support developing with the Polaris Design System.
```

Removing intellisense as that is what it does now but it might do other things in the future.